vcf/model.py

```diff
@@ -680,6 +680,14 @@ def __repr__(self):
         return str(self)
 
 
+def create_call_data(fields_dict, types, nums):
```

Comment:
please rename this. first of all to start from `_` because it is internal. also maybe more semantic:

`_restore_calldata_instance`